build.sbt

```diff
@@ -117,6 +117,29 @@ lazy val root = (project in file("."))
     )
   )
 
+lazy val scalaVersionDependentSettings = Def.settings(
+  libraryDependencies ++= (if (scalaVersion.value.startsWith("2."))
+                             COMPILE(
+                               scalaVersion("org.scala-lang" % "scala-reflect" % _).value
+                             )
+                           else Seq.empty),
```

Comment:
I would close the the parenthesis in the next line :)

Reply:
Done 👍